incubator/hnc/api/v1alpha2/hierarchy_types_conversion.go

```diff
@@ -0,0 +1,28 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
```

Comment:
According to kubebuilder [example](https://kubebuilder.io/multiversion-tutorial/conversion.html), it says "We’ll put them in a file called cronjob_conversion.go next to our cronjob_types.go file, to avoid cluttering up our main types file with extra functions." and later says "We could also just put this inline in our cronjob_types.go file.". So it's okay to do it in a separate file or inline and I just followed the example.

Reply:
sgtm